src/community/china.md

```diff
@@ -6,74 +6,171 @@ toc: true
+   ```
+
+   Remember to reset this environment variable after you publish.
+
+   ```terminal
+   export PUB_HOSTED_URL=https://pub.flutter-io.cn
+   ```
+
+1. Configure a proxy.
+   To configure a proxy, check out the [Dart documentation on proxies][].
+
+  {% comment %}
+  From https://github.com/flutter/website/issues/2556#issuecomment-481566476
+  {% endcomment %}
+
+1. Enable your VPN without adding a `credentials.json` file.
+
+1. Disable your proxy and add a `credentials.json` file.
+
+[Dart documentation on proxies]: {{site.dart-site}}/tools/pub/troubleshoot#pub-get-fails-from-behind-a-corporate-firewall
 
-After these steps, you should be able to continue
-[setting up Flutter]({{site.url}}/get-started/editor) normally.
-From here on, packages fetched by `flutter pub get` are
-downloaded from `flutter-io.cn` in any shell where `PUB_HOSTED_URL`
-and `FLUTTER_STORAGE_BASE_URL` are set.
+## Details on mirror sites
+
+### Support for mirror sites
+
+The Flutter team cannot guarantee long-term availability of any mirrors.
+You can use other mirrors if they become available.
+
+### New mirror sites in China
 
-The `flutter-io.cn` server is a provisional mirror for Flutter
-dependencies and packages maintained by [The China Flutter User Group (CFUG)][].
-The Flutter team cannot guarantee long-term availability of this service.
-You're free to use other mirrors if they become available.
 If you're interested in setting up your own mirror in China,
 contact [flutter-dev@googlegroups.com](mailto:flutter-dev@googlegroups.com)
 for assistance.
 
-If you're running into issues that only occur when using the `flutter-io.cn` server,
-consider reporting the issue to
-[the issue tracker (镜像问题)](https://github.com/cfug/flutter.cn/issues/new/choose).
+### Known, trusted community-run mirror sites
+
+#### China Flutter User Group
 
-## Community-run mirror sites
+[China Flutter User Group][] maintains the `flutter-io.cn` mirror.
+It includes Flutter and packages.
 
-* Shanghai Jiao Tong University Linux User Group
-  * `FLUTTER_STORAGE_BASE_URL`: [https://mirror.sjtu.edu.cn/](https://mirror.sjtu.edu.cn)
-  * `PUB_HOSTED_URL`: [https://mirror.sjtu.edu.cn/dart-pub/](https://mirror.sjtu.edu.cn/dart-pub)
+If you're running into issues that only occur when
+using the `flutter-io.cn` mirror, report the issue to their
+[issue tracker (镜像问题)]({{site.github}}/cfug/flutter.cn/issues/new/choose).
+
+To set your machine to use this mirror, use these commands:
+
+```bash
+export PUB_HOSTED_URL=https://pub.flutter-io.cn
+export FLUTTER_STORAGE_BASE_URL=https://storage.flutter-io.cn
+```
+
+#### Shanghai Jiao Tong University *nix User Group
+
+[Shanghai Jiao Tong University *nix User Group][]
+maintains the `mirror.sjtu.edu.cn` mirror.
+It includes Flutter and packages.
+
+If you're running into issues that only occur when
+using the `mirror.sjtu.edu.cn` mirror, report the issue to their
+[issue tracker (议题)](https://git.sjtu.edu.cn/sjtug/flutter-sdk/-/issues).
```

Comment:
we track our issues at https://github.com/sjtug/mirror-requests instead of git.sjtu.edu.cn